torch/csrc/autograd/python_variable.cpp

```diff
@@ -726,8 +726,14 @@ static PyObject* THPVariable_make_wrapper_subclass(
         size_bytes,
         /*allocator=*/c10::GetAllocator(c10::kMeta),
         /*resizable=*/true};
-    // TODO: constructor should probably accept data pointer
```

Comment:
Given that there's a TODO here I assume this is something we want to do? cc @albanD @ezyang I'm not quite sure the consequence if we expose this as a arg to tensor wrapper subclass